src/components/rockets/rockets.js

```diff
@@ -1,21 +1,57 @@
 import React from 'react';
-import logoImage from '../../images/planet.png';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
 import './rockets.css';
+import { reserveRocket } from './redux/rockets';
+
+const RocketCard = (props) => {
+  const dispatch = useDispatch();
+  const {
+    rocketName, description, flickrImages, reserved, id,
+  } = props;
+
+  const handleRocketReserve = (rocketId) => {
+    dispatch(reserveRocket(rocketId));
+  };
+
+  const reserveButtons = !reserved ? (
+    <button type="button" className="rocket-button" onClick={() => handleRocketReserve(id)}>
+      Reserve Rocket
+    </button>
+  )
+    : (
+      <button
+        type="button"
+        onClick={() => handleRocketReserve(id)}
+        className="rocket-cancel"
+      >
+        Cancel Reservation
+      </button>
+    );
 
-export default function RocketCard() {
   return (
     <div className="rocket-card">
-      <img src={logoImage} width={70} height={70} alt="planet-logo" />
+      <img src={flickrImages} width={200} height={200} alt={rocketName} />
       <div className="rocket-card-content">
-        <h1> Falcon 1 </h1>
-        <p>
-          September 2008, Falcon 1 became the first privately-developed
-          liquid-fuel launch vehicle to go into orbit around the Earth.
-          Reserve Rocket 1 was the first privately-developed liquid-fuel rocket to
-          be launched into orbit.
-        </p>
-        <button type="button" className="rocket-button">Reserve Rocket</button>
+        <h1>
```

Comment:
- [ ] For you to achieve the required design, consider adding styling for the `h1` holding the rocket name/title removing `margin-left`, or defaulting to zero does the trick.

take a look at the below images.
### Required design

![emmanuels](https://user-images.githubusercontent.com/47344665/162036119-9df06bca-92ce-4b54-a5ad-b6a9cd5f8d0d.png)

### Current design

![current](https://user-images.githubusercontent.com/47344665/162036185-a4a24b69-6191-4c92-a9b7-7e131c6a56c2.png)

Reply:
@Bavon101, These changes have been made I'll push it to the next PR.